project/ntut-xuan/README.rst

```diff
@@ -0,0 +1,163 @@
+PolygonDust - An application to calcualate the complex polygon area.
+===============================================================================
+
+
+Description
+-------------------------------------------------------------------------------
+
+PolygonDust is an application to calculate the polygon area. 
+
+We trying to calculate the polygon with cell of grid and measure the size of 
+polygon.
+
+We support that you describe complex polygon and get the area of complex polygon. 
+
+For complex polygon, we support that you can input two or more polygon 
+to doing bitwise operator like union, difference or clip. 
+
+You can select what polygon you want to doing with the pervious polygon.
+
+.. image:: https://media.discordapp.net/attachments/950048467294760990/1284862080050135071/image.png?ex=66e82c8f&is=66e6db0f&hm=8f52c071f46be06d7d6d014e8551087f495e2cf1332a784079c4805429e7aea3&=&format=webp&quality=lossless&width=2880&height=848
+
+
+How do we describe the polygon
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We will trying to describe the polygon with point set. 
```

Comment:
If there is a reference for the term "point set", could you please add it in the proposal?  A set is sometimes considered unordered in math and programming languages, but a polygon may need an ordered list.  Making it explicit will help coding.